### DISCUSSION TITLE
[question] How to get the last updated time of a directory?

### DISCUSSION THREAD

**Discussion by MarkRushB** [python-gitlab/python-gitlab#2530] [Q&A] [answered]
## Description of the problem, including code/CLI snippet
for example, the repo structure is like:

      root
      - dir1
        - file1.py
      - dir2
        - file2.py  

how can I get the latest update time of whole dir1?

## Specifications

  - python-gitlab version:
  - API version you are using (v3/v4):
  - Gitlab server version (or gitlab.com):

**Comment by nejch** [comment, marked as answer]
Unfortunately git doesn't work with directories but files. To my knowledge, best you can do is get the repository tree for that directory and then inspect the blame for individual files in there, at least via the REST API that we use.
